docs/user/gke-sre.md

```diff
@@ -34,6 +34,30 @@ these failures.
 - If the issue seems to be related to the third party tool, file a bug to the
   associated repo else file one for the Turbinia team.
 
+### Turbinia Controller
+
+In addition to the troubleshooting steps above, you may also consider deploying
+the Turbinia controller to the GKE cluster for further troubleshooting. The
+controller pod has the Turbinia client installed and is configured to use your
+Turbinia GKE instance. You may create Turbinia requests from this pod to process
+GCP disks within your project as well as have access to all Turbinia logs and output
+stored in the Filestore path. To deploy the Turbinia controller, please take the following steps.
```

Comment:
Maybe mention that this will also deploy the rest of the infrastructure as well since people might assume this command will only deploy the controller.

Reply:
Ack. added a note below stating that ^^